py3status/modules/insync.py

```diff
@@ -47,17 +47,27 @@ def insync(self):
                     'color': self.py3.COLOR_BAD,
                     'full_text': STRING_UNAVAILABLE}
 
-        # get sync progress
-        queued = self.py3.command_output(["insync", "get_sync_progress"]).splitlines()
+        # sync progress
+        try:
+            queued = self.py3.command_output(["insync", "get_sync_progress"]).splitlines()
+        except:
+            return {'cache_until': self.py3.CACHE_FOREVER,
```

Comment:
This is rare/strange exception. I ran that command just now... Got `Insync doesn't seem to be running. Start it first.` with no daemon running. I'm guessing I had insync daemon running at that time to get `Connection refused` but can't confirm anything.

I'm also wondering if running `insync get_sync_progress` (and that other command) will ping Insync service and that we did this every 10 seconds. I can't reproduce this issue easily so I think it's a little break from WTF just happened unexpectedly.